portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessdeliveryapi/CommentAPI.macro

```diff
@@ -68,7 +116,14 @@ definition {
 			var parentCommentId = "0";
 		}
 
-		var curl = CommentAPI._curlBlogPostingComments(blogPostingId = "${blogPostingId}");
+		if (isSet(blogPostingId) && !(isSet(commentId))) {
```

Comment:
Understand that only one of these parameters should be set, how about just using:
```
if(isSet(blogPostingId)){
}
if(isSet(commentId)){
}
```